.yamllint

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Looks like this is not actually taken into account. Please, refer to my previous comments for example if calling yamllint via pre-commit.

Reply:
Applied yamllint and fixed all issues following your repos.